server/runtime_go_nakama.go

```diff
@@ -2478,3 +2478,30 @@ func (n *RuntimeGoNakamaModule) SetEventFn(fn RuntimeEventCustomFunction) {
 	n.eventFn = fn
 	n.Unlock()
 }
+
+func (n *RuntimeGoNakamaModule) ChannelMessageSend(ctx context.Context, channelId, content, senderId, senderUsername string, persist bool) (*rtapi.ChannelMessageAck, error) {
+	channelIdToStreamResult, err := ChannelIdToStream(channelId)
+	if err != nil {
+		return nil, err
+	}
+
+	contentStr := "{}"
+	if content != "" {
+		contentBytes, err := json.Marshal(content)
```

Comment:
Is `content` here intended to be a `string` rather than a `map[string]interface{}`?